index.html

```diff
@@ -3,8 +3,58 @@
 
 <head>
   <title id="title">Crop AI</title>
+
+  <!-- SEO Tags will go here -->
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  <meta name="keywords"
+    content="Tech Event, Coding Competition, Open Source Competitions, Introduction to Open Source, Open Source for beginners, Girl Script Summer of Code, Girl Script Open Source Projects, Introduction to Programming, Introduction to Machine Learning ,Introduction to Deep Learning,Introduction to Front End Development, Learn Machine Learning, Learn Deep Learning, Learn FrontEnd Development,Machine Learning, Deep Learning, Front End Development">
+
```

Comment:
Should we add keywords related to plant/diseases/leaf..?

Reply:
Sir I believe they should be very general keywords and adding diseases would be kind of specific?

What do you think?

Reply:
Just add one "plant disease"!

Reply:
or something like that, which is better!

Reply:
Sure I'll search a good one and add it and inform you back!